OLGA/OLGA-Core/src/test/java/semanticstore/ontology/library/generator/generators/java/OLGAJavaVersionTest.java

```diff
@@ -85,7 +85,7 @@ public void testSimple4CodeGenerationMyValidVersion() throws Exception {
     inputCmdParameters.put("out", folder4Versions.toFile().toString());
 
     String resourcesDirectory = null;
-    resourcesDirectory = Paths.get(OLGAVersionTest.class.getClassLoader()
+    resourcesDirectory = Paths.get(OLGAJavaVersionTest.class.getClassLoader()
```

Comment:
Should we add "Java" at the end of the test name like "Csharp" and python?

Reply:
We can always run the Java ones since the maven is already built 
I did add Java but then they become 
XxxJava.java which a bit odd